admin/class-boldgrid-backup-admin-settings.php

```diff
@@ -700,6 +700,15 @@ private function update_settings() {
 			$settings['folder_exclusion_exclude'] = $this->core->folder_exclusion->from_post( 'exclude' );
 			$settings['folder_exclusion_type']    = $this->core->folder_exclusion->from_post( 'type' );
 
+			/*
+			 * Save Auto Backup options
+			 *
+			 * @since SINCEVERSION
+			 */
+			if ( ! empty( $_POST['auto_update'] ) ) {
+				$settings['auto_update'] = $_POST['auto_update'];
```

Comment:
I had discussed that before.....with joe I think.... There is no sanitization needed here since 1) none of the auto_update options are text input, they are all toggles, or number inputs and 2) any output created from these particular inputs are sanitized before being output

Reply:
however, this may have been something else we were discussing. If you think it is necessary, I can write some sanitization for it in the Connect::sanitizeSettings method.